### PR TITLE
Fix: Supply Stash selection sound sometimes not played

### PR DESCRIPTION
**Pull request by commy2**

### CODE REVIEW

File: Patch104pZH/Design/Changes/v1.0/2188_supply_stash_selection_sound.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+date: 2023-08-01
+
+title: Fixes Supply Stash selection sound sometimes not played
+
+changes:
+  - fix: Fixes an issue where the Supply Stash selection sound has the wrong priority.
```
Comment (xezon) [inline Patch104pZH/Design/Changes/v1.0/2188_supply_stash_selection_sound.yaml]: "Sets the correct event types on the GLA Supply Stash select audio event."

File: Patch104pZH/Design/Changes/v1.0/2188_supply_stash_selection_sound.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+---
+date: 2023-08-01
+
+title: Fixes Supply Stash selection sound sometimes not played
```
Comment (xezon) [inline Patch104pZH/Design/Changes/v1.0/2188_supply_stash_selection_sound.yaml]: "GLA Supply Stash"

**Review by xezon — APPROVED** [review]
Nice. Never noticed this.

File: Patch104pZH/Design/Changes/v1.0/2188_supply_stash_selection_sound.yaml (hunk 1 above)
Comment (commy2) [reply Patch104pZH/Design/Changes/v1.0/2188_supply_stash_selection_sound.yaml]:
```suggestion
  - fix: Fixes an issue where the GLA Supply Stash selection sound has the wrong priority.
```

File: Patch104pZH/Design/Changes/v1.0/2188_supply_stash_selection_sound.yaml (hunk 2 above)
Comment (commy2) [reply Patch104pZH/Design/Changes/v1.0/2188_supply_stash_selection_sound.yaml]:
```suggestion
title: Fixes GLA Supply Stash selection sound sometimes not played
```

**Comment by commy2** [comment]
Me neither. It's from NPM changelog. Took me a while to figure out what was meant. 
USA and China are called "Supply Center", not "Supply Stash".

**Pull request merged**